src/index.d.ts

```diff
@@ -0,0 +1,477 @@
+// Type definitions for vast-client 2.1
```

Comment:
I've checked the type definitions version and used https://github.com/DefinitelyTyped/DefinitelyTyped/blob/29216607f433cbb900acdcf4a0e4ea69e9db5376/types/vast-client/v2/index.d.ts